cumulusci.yml

```diff
@@ -288,14 +288,24 @@ flows:
         steps:
             1: 
                 task: add_application_type_picklist_values
+                ui_options:
+                    name: EDA - Application Type Picklist Values
```

Comment:
@MatthewBlanski @lifewithryan @jstvz  whats your take on EDA vs K12 for the name?

Reply:
I am all right with it in K-12 since it is explicit. You may want to check with @JasonFitzy

Reply:
I think we're using the `EDA -` prefix to denote EDA's pre/post-install steps. I think replacing the prefix with 'Add' may be clearer.